x-pack/legacy/plugins/ml/public/jobs/components/custom_url_editor/list.js

```diff
@@ -170,15 +172,24 @@ export const CustomUrlList = injectI18n(class CustomUrlList extends Component {
           </EuiFlexItem>
           <EuiFlexItem>
             <EuiFormRow
+              fullWidth={true}
               label={<FormattedMessage
                 id="xpack.ml.customUrlEditorList.urlLabel"
                 defaultMessage="URL"
               />}
             >
-              <EuiFieldText
-                value={customUrl.url_value}
-                onChange={(e) => this.onUrlValueChange(e, index)}
-              />
+              {index === expandedUrlIndex ?
+                <EuiTextArea
+                  value={customUrl.url_value}
+                  onChange={(e) => this.onUrlValueChange(e, index)}
+                  onBlur={() => this.setState({ expandedUrlIndex: null })}
```

Comment:
I've noticed that the input doesn't collapse back to a text field if you mouse click, for example, in the 'Label' field of a different custom URL. Is there anything that can be done about that, so that the field is only a text area when it has focus? Seems to work mostly as intended on tab navigation.